### ISSUE TITLE
dependancy conflicts : `pyleniumio 1.15.4` - `black 22.3.0`

### ISSUE THREAD

**Issue by lolrenx** [ElSnoMan/pyleniumio#263]
Just a heads up since black is so widely used :

```
The conflict is caused by:
    The user requested click
    black 22.3.0 depends on click>=8.0.0
    pyleniumio 1.15.4 depends on click<8.0.0 and >=7.1.2
```

**Comment by ElSnoMan** [comment]
Great catch! Thank you for pointing this out 🙏🏽

**Comment by Rizhiy** [comment]
BTW, `click<8.0.0` was introduced in `1.15.3`, so you actually need `pyleniumio<1.15.3` to fix conflict.

**Comment by ElSnoMan** [comment]
Changed requirement to `click>=7.1.2`

**Comment by ElSnoMan** [comment]
Released in v1.16.1
